frontend/app/components/Spots/SpotPlayer/components/SpotVideoContainer.tsx

```diff
@@ -1,10 +1,13 @@
 import Hls from 'hls.js';
 import { observer } from 'mobx-react-lite';
 import React from 'react';
+import {Alert, Button} from 'antd';
+import {PlayCircleOutlined, InfoCircleOutlined} from '@ant-design/icons';
 
 import { useStore } from 'App/mstore';
 
 import spotPlayerStore from '../spotPlayerStore';
+import { color } from 'html2canvas/dist/types/css/types/color';
```

Comment:
was this an accident? I think it shouldn't be here

Reply:
Yes, unintentional not sure how it got in there.